libparsec/crates/client/src/workspace/merge.rs

```diff
@@ -150,7 +150,19 @@ pub(super) fn merge_local_folder_manifest(
     prevent_sync_pattern: Option<&Regex>,
```

Comment:
Should I remove the option ?

For me I can since I ways provide a value during the `inbound_sync`

Reply:
Yes, the `prevent_sync_pattern` should now always be present

(the `Option` was added in the codebase as a way to disable prevent sync pattern so we could skip it implementation, so it's totally useless once you job is done here ^^)